### DISCUSSION TITLE
TBC XP requirements during TBC content

### DISCUSSION THREAD

**Discussion by Soulex23** [chromiecraft/chromiecraft#3958] [Suggestions]
In Wrath, the XP requirements for leveling from 60-70 is squished quite a lot to help funnel players through it to the Wrath content.

It would make a bit more sense to have the same levels of XP required for 60-70 during TBC content when those brackets unlock so that people do more quests and play more content, thereby catching more bugs along the way. It would also keep players from feeling like they're just walking through dungeons for a week then raidlogging all over again. The 60-64 bracket would have players playing their class a lot more and understanding them better, as well as searching for the easily missed nooks and crannies that might otherwise be missed by simply zooming to the next piece of content. During the 65-69 bracket, same story; more bugs caught, more content explored. Once 70 rolls around, _then_ we could go back to the Wrath squish of XP requirements, since a good majority of the playerbase would be up to 70 or so by that time, and people can focus more on the heroic dungeons and raids by that point.

**Comment by balleny** [comment]
yep, a reduced xp-rate (e.g. factor 0.6) would be appropriate
